pkgs/applications/misc/revelation/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, pkgconfig, python2Packages, gnome2, intltool }:
+
+stdenv.mkDerivation rec {
+  name = "revelation-${version}";
+  version = "0.4.14";
+
+  src = fetchurl {
+    url = https://bitbucket.org/erikg/revelation/downloads/revelation-0.4.14.tar.bz2;
+    sha256 = "2ab3d1d8bcc2f441feb58122ee6a0fe4070412228194843a180a7b1c9e910019";
+  };
+
+  patches = [ ./remove-randpool.patch ];
+
+  buildInputs = [ pkgconfig gnome2.GConf intltool ];
```

Comment:
`pkgconfig` belongs to `nativeBuildInputs`.

Reply:
intltool maybe as well?